DuckDuckGo/Subscription/Views/SubscriptionEmailView.swift

```diff
@@ -74,4 +75,12 @@ struct SubscriptionEmailView: View {
     
     
 }
+
+// @available(iOS 15.0, *)
+// struct SubscriptionEmailView_Previews: PreviewProvider {
+//    static var previews: some View {
+//        SubscriptionEmailView()
+//    }
+// }
+
```

Comment:
Do we need to comment this out?   If it gives problems in CI, you can try wrapping in #DEBUG or remove.